README.md

```diff
@@ -1,53 +1,93 @@
 
-- **Title:** Template
-- **Identifier:** <https://stac-extensions.github.io/template/v1.0.0/schema.json>
-- **Field Name Prefix:** template
+- **Title:** Classification
+- **Identifier:** <https://stac-extensions.github.io/classification/v1.0.0/schema.json>
+- **Field Name Prefix:** classification
 - **Scope:** Item, Collection
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
-- **Owner**: @your-gh-handles @person2
+- **Owner**: @drwelby @mmohr
+
+This document explains the Classification Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
+
+Classification stores metadata that clarifies the values within a dataset. Common uses would be:
+
+- Describing classes of data, and the values belonging to the class
+- The reverse of the above, as a lookup table mapping values to classes
 
-This document explains the Template Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
-This is the place to add a short introduction.
 
 - Examples:
-  - [Item example](examples/item.json): Shows the basic usage of the extension in a STAC Item
+  - [Asset example](examples/item.json): Shows the basic usage of the extension in a STAC Item
   - [Collection example](examples/collection.json): Shows the basic usage of the extension in a STAC Collection
 - [JSON Schema](json-schema/schema.json)
 - [Changelog](./CHANGELOG.md)
 
-## Item Properties and Collection Fields
+## Asset Properties 
+
+For single-band rasters
+
+## Raster Band (Raster Extension)
+
+For multiband rasters
+
+## Table Column (Table Extension)
+
+For tabular or vector datasets
+
+## Collection Fields
+
+In `summaries` field
+
+
+_naming the field "classes" seems more descriptive but ??_
 
 | Field Name           | Type                      | Description |
 | -------------------- | ------------------------- | ----------- |
-| template:new_field   | string                    | **REQUIRED**. Describe the required field... |
-| template:xyz         | [XYZ Object](#xyz-object) | Describe the field... |
-| template:another_one | \[number]                 | Describe the field... |
+| classification:classes   | [Class object]        | **REQUIRED**. Classes in the dataset |
 
-### Additional Field Information
 
-#### template:new_field
+_Current fields from file:values_
+
+### Class Object
+
+| Field Name           | Type                      | Description |
+| -------------------- | ------------------------- | ----------- |
+| values  | [Any] | **REQUIRED** Values in the class |
+| summary | string                | REQUIRED. A short description of the value(s). |
 
-This is a much more detailed description of the field `template:new_field`...
 
-### XYZ Object
+_"values" is shown as a list of any object. I think that could include Range objects for continuous data_
```

Comment:
I think this would be a useful feature to support. Could we add some specific instructions/examples on how to represent these continuous values?

Reply:
What bothers me is that putting in ranges seems like you have data that isn't "classed" in the first place. Maybe that's too strict though. Do you have an example of how you can use this?

Reply:
If you could also give an example of your ML label rasters in https://github.com/stac-extensions/classification/issues/1#issuecomment-932489887 that would help out.

Reply:
Yeah, that's a fair point.

I don't have an example at hand, but I know I've worked with land cover rasters in the past where a particular class was represented by a value range for one reason or another. However, they were usually limited to 2-3 values, so we could reasonably handle that using a list.

Reply:
Yup, I've seen LC coverages that have classes that get merged, like "Scrub Woodland", "Deciduous Woodland", etc, and they get flattened to "Woodland". So you're reclassing the data after the fact, and that seems interpretive and not descriptive. Repeating classes in a list doesn't hide the original data intent as much and might convince people to properly class their values?

I mostly want to avoid having data that is really continuous (like a confidence percentage) and someone saying "_yeaahhh, 75% confidence sounds good enough for these values to describe a feature_"